isc_dhcp_leases/iscdhcpleases.py

```diff
@@ -160,6 +162,9 @@ def __init__(self, ip, data, cltt, host_identifier, address_type):
         self.last_communication = cltt
 
         self.host_identifier = codecs.decode(host_identifier, "unicode_escape")
+        host_identifier_bytes = self.host_identifier.encode('latin-1')
```

Comment:
The original host_identifier string can't be used directly the unicode_escape decode is required first.
The latin-1 encode is required as we can't convert unicode directory to hex